File: src/app/model/Supplier.ts
export class Supplier 
{
    id: number; 
    name: string; 
    contactPerson: string; 
    email: string; 
    phone: any; 
    location:string;
    assetName:string;
    
  
    constructor(id: number, name: string, contactPerson: string, email: string, phone: string, location:string,assetName:string)
     {
      this.id = id;
      this.name = name;
      this.contactPerson = contactPerson;
      this.email = email;
      this.phone = phone;
      this.location = location;
      this.assetName=assetName;
     

    }
  }
  

File: Spring/SpringBoot/finalProject/src/main/java/edu/training/finalProject/model/Supplier.java
package edu.training.finalProject.model;

import java.util.List;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.ManyToOne;
import jakarta.persistence.OneToMany;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
public class Supplier 
{
	
  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  int id; 
  String name; 
  String contactPerson; 
  String email; 
  String phone; 
  String location;
  String assetName;

}
